### PR TITLE
Release notes for AWS Load Balancer Operator

### PR DESCRIPTION
**Pull request by xenolinux**
Release notes for AWS Load Balancer Operator 

Version(s): 4.11

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/RN-ALB-OP/release_notes/ocp-4-11-release-notes.html#ocp-4-11-alb-operator-support

Additional information:

### CODE REVIEW

**Comment by lihongan** [comment]
It looks good. 
Thank you @xenolinux

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -163,6 +163,13 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
 
 For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy[Service Internal Traffic Policy].
 
+[id="ocp-4-11-alb-operator-support"]
+==== Support for AWS Load Balancer Operator (TP)
```
Comment (darshan-nagaraj) [inline release_notes/ocp-4-11-release-notes.adoc:167]:
```suggestion
==== Support for AWS Load Balancer Operator (Technology Preview)
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -163,6 +163,13 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
...
+
+As a cluster administrator, you can install AWS Load Balancer Operator from the OperatorHub by using {product-title} web console or CLI. The AWS Load Balancer Operator is in technology preview.
```
Comment (darshan-nagaraj) [inline release_notes/ocp-4-11-release-notes.adoc:169]:
```suggestion
As a cluster administrator, you can install the AWS Load Balancer Operator from OperatorHub using the {product-title} web console or CLI. The AWS Load Balancer Operator is available as a Technology Preview.
```

Comment (darshan-nagaraj) [reply release_notes/ocp-4-11-release-notes.adoc:169]: I think you also need to add the AWS Load Balancer Operator in Technology Preview tracker tabel.
 https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

**Review by darshan-nagaraj — COMMENTED** [review]
Few nits inline. Let me know if you've any questions.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (xenolinux) [reply release_notes/ocp-4-11-release-notes.adoc:169]: @darshan-nagaraj IMO, The Operator is not a technology preview, but it has reached TP status. Similarly, it is not a Technology Preview, but it is in TP _status_.

Comment (xenolinux) [reply release_notes/ocp-4-11-release-notes.adoc]: Addressed other suggestions, thanks for the review

Comment (xenolinux) [reply release_notes/ocp-4-11-release-notes.adoc]: > I think you also need to add the AWS Load Balancer Operator in Technology Preview tracker tabel. https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

Addressed

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -392,6 +392,13 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
 
 For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy[Service Internal Traffic Policy].
 
+[id="ocp-4-11-alb-operator-support"]
+==== Support for AWS Load Balancer Operator (Technology Preview)
+
+As a cluster administrator, you can install the AWS Load Balancer Operator from the OperatorHub using the {product-title} web console or CLI. The AWS Load Balancer Operator is in technology preview.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-11-release-notes.adoc:398]:
```suggestion
As a cluster administrator, you can install the AWS Load Balancer Operator from the OperatorHub using the {product-title} web console or CLI. The AWS Load Balancer Operator is in Technology Preview.
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -392,6 +392,13 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
...
+
+For more information, see  xref:../networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc#aws-load-balancer-operator[Installing AWS Load Balancer Operator]
```
Comment (bergerhoffer) [inline release_notes/ocp-4-11-release-notes.adoc:400]:
```suggestion
For more information, see xref:../networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc#aws-load-balancer-operator[Installing AWS Load Balancer Operator].
```

Comment (xenolinux) [reply release_notes/ocp-4-11-release-notes.adoc]: Addressed

File: release_notes/ocp-4-11-release-notes.adoc (hunk 3 above)
Comment (xenolinux) [reply release_notes/ocp-4-11-release-notes.adoc]: Ack'ed

**Comment by bergerhoffer** [comment]
LGTM now, thanks, merging!

**Pull request merged**